### DISCUSSION TITLE
umi/max 开启mako，react-player编译报错

### DISCUSSION THREAD

**Discussion by Ty-hongxitai** [umijs/umi#12427] [Q&A]
> @ build /demo
> max build

info  - [你知道吗？] dev 模式下访问 /__umi 路由，可以发现很多有用的内部信息。
info  - [plugin: ./node_modules/.pnpm/@umijs+preset-umi@4.2.8_@types+node@20.12.13_@types+react@18.3.3_lightningcss@1.22.1_rollup@3_x4cfsshlp5urnn3pybxej73rhq/node_modules/@umijs/preset-umi/dist/features/mako/mako] Using mako@0.5.2
warn  - [plugin: ./node_modules/.pnpm/@umijs+preset-umi@4.2.8_@types+node@20.12.13_@types+react@18.3.3_lightningcss@1.22.1_rollup@3_x4cfsshlp5urnn3pybxej73rhq/node_modules/@umijs/preset-umi/dist/features/mako/mako] Mako is an extremely fast, production-grade web bundler based on Rust. And it's still under active development and is not yet ready for production use. If you encounter any issues, please checkout https://makojs.dev/ to join the community and report the issue.
info  - Umi v4.2.8
info  - Preparing...
Building with mako for production...
failed to resolve ?context&glob=**/*&async from /demo/node_modules/.pnpm/react-player@2.16.0_react@18.1.0/node_modules/react-player/lib/players/Mux.js with resolver err: Specifier(Empty("?context&glob=**/*&async"))
Build failed.

  × Module not found: Can't resolve '?context&glob=**/*&async'

fatal - [Error: Build with mako failed.]
fatal - A complete log of this run can be found in:
fatal - /demo/node_modules/.cache/logger/umi.log
fatal - Consider reporting a GitHub issue on https://github.com/umijs/umi/issues
fatal - 如果你需要进交流群，请访问 https://fb.umijs.org 。

**Comment by fz6m** [comment]
mako 现在没有开源，如有问题，在 [这篇文章](https://makojs.dev/blog/mako-internal-test) 、云谦的公众号 等地方寻找途经反馈即可。

**Comment by YJKZHUZHU** [comment]
同样遇到一样的问题
